emscripten.py

```diff
@@ -796,6 +796,10 @@ def make_export_wrappers(exports, delay_assignment):
     if name == '__cpp_exception':
       continue
     mangled = asmjs_mangle(name)
+    if mangled in building.user_requested_exports or name.startswith('dynCall_'):
```

Comment:
Why are dyncalls special? (my only guess is dynamic linking..?)

Reply:
Ah yes, its because the references to these are dynamically constructed at runtime and expected to exist on the Module object (the internal versions are not useful here):

https://github.com/emscripten-core/emscripten/blob/main/src/library.js#L3205

Reply:
I will add a comment